standard/expressions.md

```diff
@@ -2819,17 +2819,28 @@ The `unchecked` operator is convenient when writing constants of the signed inte
 
 ### 11.7.19 Default value expressions
 
-A default value expression is used to obtain the default value ([§9.3](variables.md#93-default-values)) of a type. Typically a default value expression is used for type parameters, since it might not be known if the type parameter is a value type or a reference type. (No conversion exists from the `null` literal ([§6.4.5.7](lexical-structure.md#6457-the-null-literal)) to a type parameter unless the type parameter is known to be a reference type ([§8.2](types.md#82-reference-types)).)
+A default value expression is used to obtain the default value ([§9.3](variables.md#93-default-values)) of an explicit or inferred type. Typically, a default value expression is used for type parameters, since it might not be known if the type parameter is a value type or a reference type. (No conversion exists from the `null` literal ([§6.4.5.7](lexical-structure.md#6457-the-null-literal)) to a type parameter unless the type parameter is known to be a reference type ([§8.2](types.md#82-reference-types)).)
 
 ```ANTLR
 default_value_expression
+    : explictly_typed_default
+    | default_literal
+    ;
+
+explictly_typed_default
     : 'default' '(' type ')'
     ;
+
+default_literal
+    : 'default'
+    ;
 ```
 
-If the *type* in a *default_value_expression* evaluates at run-time to a reference type, the result is `null` converted to that type. If the *type* in a *default_value_expression* evaluates at run-time to a value type, the result is the value type’s default value ([§8.3.3](types.md#833-default-constructors)).
+*default_literal* allows the type to be inferred by target-typing.
```

Comment:
Agree with @jskeet, "target-typing" is not used in the Standard. I think we should describe *default_literal* in the same way as a *null_literal* is described, i.e. replace this line with:

> A `default_literal` represents a default value (§9.3). It does not have a type, but can be converted to any type
or nullable through a default literal conversion (§default-literal-new-clause)

Reply:
That looks good to me. Will discuss at the meeting later "today" (UTC). @Nigel-Ecma: I know you can't make the meeting, but I'm going to assume you're happy with your own suggestion.

Reply:
@jskeet – in this rare case I do agree with myself ;-)